apps-rendering/src/components/TableOfContent/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { css, SerializedStyles } from '@emotion/react';
+import { text } from '@guardian/common-rendering/src/editorialPalette/text';
+import { ArticleFormat } from '@guardian/libs';
+import {
+	line,
+	neutral,
+	remSpace,
+	textSans,
+} from '@guardian/source-foundations';
+import {
+	SvgChevronDownSingle,
+	SvgChevronUpSingle,
+} from '@guardian/source-react-components';
+import Anchor from 'components/Anchor';
+import ListItem from 'components/ListItem';
+import OrderedList from 'components/OrderedList';
+import { Outline } from 'outline';
+import { FC } from 'react';
+import { darkModeCss } from 'styles';
+
+interface Props {
+	format: ArticleFormat;
+	outline: Outline;
+}
+
+const anchorStyles = (format: ArticleFormat): SerializedStyles => css`
+	color: ${text.paragraph(format)};
+	border-bottom: none;
+	:hover {
+		border-bottom: 0.0625rem solid ${neutral[86]};
+	}
+`;
+
+const listStyles: SerializedStyles = css`
+	> li::before {
+		content: none;
+	}
+
+	margin: 0;
+
+	> li {
+		padding-left: ${remSpace[3]};
+	}
+`;
+
+const listItemStyles: SerializedStyles = css`
+	${textSans.xsmall({ fontWeight: 'bold', lineHeight: 'regular' })}
+	border-bottom: ${line.primary} 1px solid;
+	padding-top: ${remSpace[2]};
+`;
+
+const detailsStyles: SerializedStyles = css`
+	margin-bottom: 1.25rem;
+	&:not([open]) .is-on,
+	&[open] .is-off {
+		display: none;
+	}
+	summary::-webkit-details-marker {
+		display: none;
+	}
+`;
+
+const summaryStyles: SerializedStyles = css`
+	cursor: pointer;
+	position: relative;
+	list-style: none;
+	align-items: center;
+	padding-left: ${remSpace[3]};
+	padding-top: 0.44rem;
+	padding-bottom: 0.375rem;
+	border-bottom: ${line.primary} 1px solid;
+	border-top: ${line.primary} 1px solid;
+
+	path {
+		fill: ${neutral[46]};
+	}
+	svg {
+		height: 2rem;
+	}
+`;
+
+const titleStyle: SerializedStyles = css`
+	${textSans.xsmall({ lineHeight: 'regular' })}
+	color: ${neutral[46]};
+	${darkModeCss`
+		color: ${neutral[86]};
```

Comment:
Could these live in the `text` palette?

Reply:
I added `tableOfContentsTitle` & `tableOfContentsTitleDark` in the `text` pallet